wazo_dird/plugins/google_backend/plugin.py

```diff
@@ -121,9 +121,15 @@ def _first_match_predicate(self, term, contact):
             column_value = contact.get(column) or ''
             if isinstance(column_value, (dict, list)):
                 for value in column_value:
-                    if term == value.lower():
-                        return True
+                    if isinstance(value, dict):
+                        for sub_value in value.values():
+                            if term == sub_value.lower():
+                                return True
+                    else:
+                        if term == value.lower():
+                            return True
             else:
+                logger.debug('Item is not a dict or list')
```

Comment:
We can remove it.
But just wait until zuul complete its check and we will force push/merge if need to be in this release

Reply:
Oops, did not mean to leave it in. I will force push without it.